### PR TITLE
Benchmarks publication process changes

### PR DESCRIPTION
**Pull request by Sammers21**
Benchmark publication process has been changed in accordance with the latest changes in the benchmarking repository.

Related to #431 

Those changes were tested on my fork repository.

### CODE REVIEW

**Comment by Sammers21** [comment]
@g4s8, review the PR please.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -35,17 +35,23 @@ jobs:
...
-        uses: JamesIves/github-pages-deploy-action@releases/v3
+      - name: Commit benchmark results
+        run: |
+          export REPORT=${{ steps.run_benchmarks.outputs.report }}
+          export VERSION=${{ env.RELEASE_VERSION }}
+          git checkout gh-pages
+          mkdir -p benchmarks/$VERSION
+          mv $REPORT benchmarks/$VERSION/$REPORT
+          git add benchmarks/$VERSION/$REPORT
+          git commit -m "Add benchmark results"
```
Comment (g4s8) [inline .github/workflows/release.yml:52]: @Sammers21 please add `$VERSION` name to commit message, e.g. `Add benchmark results for $VERSION`

**Comment by Sammers21** [comment]
@g4s8, can we merge?

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#437` is not assigned, can't get performer

**Pull request merged**